From 512e9845a53f1e00d3f8d5746e884bbc18f12f13 Mon Sep 17 00:00:00 2001
From: Dmitri Epshtein <dima@marvell.com>
Date: Sat, 28 May 2016 21:25:31 +0300
Subject: [PATCH 472/538] mvneta: Use napi_alloc_skb()

- This change replaces calls to netdev_alloc_skb_ip_align with
napi_alloc_skb as done for most other drivers

Change-Id: I7b2f8103dc5323658cd76f28f46d3d2b05182e85
Signed-off-by: Dmitri Epshtein <dima@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30084
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-by: Lior Amsalem <alior@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/net/ethernet/marvell/mvneta.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/mvneta.c b/drivers/net/ethernet/marvell/mvneta.c
index 36aebaf..e6ab9d8 100644
--- a/drivers/net/ethernet/marvell/mvneta.c
+++ b/drivers/net/ethernet/marvell/mvneta.c
@@ -1792,7 +1792,7 @@ err_drop_frame:
 
 		if (rx_bytes <= rx_copybreak) {
 			/* better copy a small frame and not unmap the DMA region */
-			skb = netdev_alloc_skb_ip_align(dev, rx_bytes);
+			skb = napi_alloc_skb(napi, rx_bytes);
 			if (unlikely(!skb)) {
 				netdev_warn(dev, "rxq #%d - Can't allocate skb. rx_bytes = %d bytes\n",
 					    rxq->id, rx_bytes);
-- 
1.9.1

